components/table/SearchFilterSort.tsx

```diff
@@ -134,6 +133,10 @@ export default function SearchFilterSort({
                     onBlur={() => setFocused(false)}
                     onChange={(e) => setSearchTerm(e.target.value)}
                     spellCheck={false}
+                    onKeyDown={(e) => {
+                      //prevent page reboot on "Enter" key press
```

Comment:
I believe we have a comment convention to start with empty character followed by a capital word;
so can we change it to `// Prevent page reboot on "Enter" key press`?
I will look for a `eslint` plugin to set it as a eslint rules.

Reply:
done